File: src/main/java/za/ac/sun/grapl/domain/stack/operand/OperatorItem.java
package za.ac.sun.grapl.domain.stack.operand;

import za.ac.sun.grapl.domain.stack.OperandItem;

public final class OperatorItem extends OperandItem {

    public OperatorItem(final String id, final String type) {
        super(id, type);
    }

    @Override
    public String toString() {
        return "OPERATOR {value: " + id + ", type: " + type + "}";
    }

}

File: src/main/java/za/ac/sun/grapl/domain/stack/operand/OperatorItem.kt
package za.ac.sun.grapl.domain.stack.operand

import za.ac.sun.grapl.domain.stack.OperandItem

class OperatorItem(id: String?, type: String?) : OperandItem(id!!, type!!) {
    override fun toString(): String {
        return "OPERATOR {value: $id, type: $type}"
    }
}